landlab/components/hybrid_alluvium/hybrid_alluvium.py

```diff
@@ -2,6 +2,15 @@
 from landlab import Component
 from .cfuncs import calculate_qs_in
 
+def rock_alluv_func(x, H0, d, s, a, etad, Hstar, R0, r):
+    """Functions of x0 (H) and x1 (R) that should evaluate to zero."""
```

Comment:
Add a proper docstring with parameter descriptions. Better parameter names and PEP8 naming conventions.